docs/book/src/reference/jobs.md

```diff
@@ -0,0 +1,79 @@
+# Jobs
+
+This document intents to provide an overview over our jobs running via Prow, GitHub actions and Google Cloud Build.
+
+## Builds and Tests running on the default branch (currently v1alpha4)
+
+> NOTE: To see which test jobs execute which tests or e2e tests, you can click on the links which lead to the respective test overviews in testgrid.
+
+### Presubmits
+
+Prow Presubmits:
+* [pull-cluster-api-build-main] ./scripts/ci-build.sh
+* \[opt\] \[always-run:false\] [pull-cluster-api-make-main] ./scripts/ci-make.sh
+* \[opt\] [pull-cluster-api-apidiff-main] ./scripts/ci-apidiff.sh
+* [pull-cluster-api-verify] ./scripts/ci-verify.sh
+* [pull-cluster-api-test-main] ./scripts/ci-test.sh
+* [pull-cluster-api-e2e-main] ./scripts/ci-e2e.sh
+  * GINKGO_FOCUS: `[PR-Blocking]`
+* \[opt\] [pull-cluster-api-e2e-ipv6-main] ./scripts/ci-e2e.sh
+  * GINKGO_FOCUS: `[PR-Blocking]`, IP_FAMILY: `IPv6`
+* \[opt\] \[always-run:false\] [pull-cluster-api-e2e-full-main] ./scripts/ci-e2e.sh
+  * GINKGO_SKIP: `[PR-Blocking] [Conformance] [K8s-Upgrade]` (i.e. "no tags")
+* \[opt\] \[always-run:false\] [pull-cluster-api-e2e-workload-upgrade-1-21-latest-main] ./scripts/ci-e2e.sh FROM: stable-1.21 TO: ci/latest-1.22
+  * GINKGO_FOCUS: `[K8s-Upgrade]`
+
+> NOTE: [opt]: jobs that don't have to be run successfully for merge
+
+> NOTE: [always-run:false]: jobs that are not triggered automatically for every commit
+
+Github Presubmit Workflows:
+* golangci-lint: golangci/golangci-lint-action@v2
+* verify: kubernetes-sigs/kubebuilder-release-tools@v0.1 verifier
+
+### Postsubmits
+
+Prow Postsubmits:
+* [post-cluster-api-push-images] Google Cloud Build: make release-staging, make -C test/infrastructure/docker release-staging
+
+### Periodics
+
+Prow Periodics:
+* [periodic-cluster-api-verify-book-links-main] make verify-book-links
+* [periodic-cluster-api-test-main] ./scripts/ci-test.sh
+* [periodic-cluster-api-e2e-main] ./scripts/ci-e2e.sh
+  * GINKGO_SKIP: `[Conformance] [K8s-Upgrade]`
+* [periodic-cluster-api-e2e-workload-upgrade-1-18-1-19-main] ./scripts/ci-e2e.sh FROM: stable-1.18 TO: stable-1.19
+  * GINKGO_FOCUS: `[K8s-Upgrade]`
+* [periodic-cluster-api-e2e-workload-upgrade-1-19-1-20-main] ./scripts/ci-e2e.sh FROM: stable-1.19 TO: stable-1.20
+  * GINKGO_FOCUS: `[K8s-Upgrade]`
+* [periodic-cluster-api-e2e-workload-upgrade-1-20-1-21-main] ./scripts/ci-e2e.sh FROM: stable-1.20 TO: stable-1.21
+  * GINKGO_FOCUS: `[K8s-Upgrade]`
+* [periodic-cluster-api-e2e-workload-upgrade-1-21-latest-main] ./scripts/ci-e2e.sh FROM: stable-1.21 TO: ci/latest-1.22
+  * GINKGO_FOCUS: `[K8s-Upgrade]`
+* [cluster-api-push-images-nightly] Google Cloud Build: make release-staging-nightly, make -C test/infrastructure/docker release-staging-nightly
```

Comment:
Might be nice to format the `make` calls to highlight that they are something that can be called on the command line.

Reply:
Done (also for the script calls)